packages/calling/calling/src/CallingClient/line/index.ts

```diff
@@ -0,0 +1,238 @@
+import {Mutex} from 'async-mutex';
+import {v4 as uuid} from 'uuid';
+import {IDeviceInfo, MobiusDeviceId, MobiusStatus, ServiceIndicator} from '../../common/types';
+import {ILine, LINE_EVENTS, LineEventTypes, LineStatus} from './types';
+import {LINE_FILE, KEEPALIVE_UTIL} from '../constants';
+import log from '../../Logger';
+import {IRegistration} from '../registration/types';
+import {createRegistration} from '../registration';
+import {ISDKConnector, WebexSDK} from '../../SDKConnector/types';
+import {SdkConfig} from '../types';
+import {Eventing} from '../../Events/impl';
+import {LineError} from '../../Errors/catalog/LineError';
+import {LOGGER} from '../../Logger/types';
+import {validateServiceData} from '../../common';
+import SDKConnector from '../../SDKConnector';
+import {EVENT_KEYS} from '../../Events/types';
+import {CallingClientError} from '../../Errors';
+
+export default class Line extends Eventing<LineEventTypes> implements ILine {
+  #webex: WebexSDK;
+
+  #mutex: Mutex;
+
+  #sdkConfig: SdkConfig | undefined;
+
+  #sdkConnector: ISDKConnector;
+
+  public registration: IRegistration;
+
+  public userId: string;
+
+  public clientDeviceUri: string;
+
+  public lineId: string;
+
+  public mobiusDeviceId: string | undefined;
+
+  public mobiusUri: string | undefined;
```

Comment:
@meta-dipanshu-sharma @Kesari3008  some of these values can be with _. if they are only used internally , for developer mobiusUri is not needed

Reply:
@Kesari3008  it might be needed for the web client for something else but not needed for developer to know correct ?

we could hide this based on type we expose if we don't want to expose to developers

Reply:
As per discussion, made `mobiusUri` as private property.